c10/core/DispatchKey.h

```diff
@@ -331,7 +332,7 @@ constexpr DispatchKey kAutograd = DispatchKey::Autograd;
 
 // Check if a DispatchKey is an alias mapping to other runtime keys.
 inline bool isAliasDispatchKey(DispatchKey k) {
-  return k == DispatchKey::Autograd;
+  return k == DispatchKey::Autograd || k == DispatchKey::Math;
```

Comment:
How about `k > NumDispatchKeys && k <= EndOfAliasKeys`? Then it'll track additions automatically